doc/common_tools_writing_lints.md

```diff
@@ -102,8 +103,7 @@ impl LateLintPass<'_> for MyStructLint {
 ```
 
 > Prefer using lang items, if the target trait is available there.
-
-A list of defined paths for Clippy can be found in [paths.rs][paths]
+> Note that using `is_trait_method` requires checking each expression on sequentials.
```

Comment:
Correct me if this is wrong, but this is not specific to `is_trait_method` but just a result of sequential calls, right? In that case, maybe something like this:
```suggestion
> Note that linting a sequence of trait calls requires checking each expression for the trait.
```

Reply:
I'd say that we do not need to check each expression, but rather it depends on the context?

After re-reading this, this seems to give more confusion rather than clarity. I suppose we can omit this one, what do you think @xFrednet

Reply:
I agree! So, sounds good to me :+1: